apps/condo/domains/common/hooks/useDownloadFileFromServer.ts

```diff
@@ -0,0 +1,65 @@
+import { useCallback } from 'react'
+import { notification } from 'antd'
+
+import { useIntl } from '@open-condo/next/intl'
+
+
+type FileType = {
+    url: string
+    name: string
+}
+type DownloadFileType = (file: FileType) => Promise<void>
+type UseDownloadFileFromServerType = () => { downloadFile: DownloadFileType }
+
+const ERROR_DOWNLOAD_FILE = 'Failed to download file'
+const DIRECT_DOWNLOAD_FILE_TYPES_REGEX = /.*\.(svg|html|htm|xml|txt)$/i
```

Comment:
Should pdf be presented in this list?

Reply:
At first, that's what I wanted to do. But then I thought that the default browser action is also a good option.
Therefore, it turned out that this functionality was simply moved to a separate hook